src/app/util/ember-compatibility-functions.cpp

```diff
@@ -408,6 +405,14 @@ CHIP_ERROR GlobalAttributeReader::Read(const ConcreteReadAttributePath & aPath,
             }
             return CHIP_NO_ERROR;
         });
+    case EventList::Id:
+        return aEncoder.EncodeList([this](const auto & encoder) {
+            for (size_t i = 0; i < mCluster->eventCount; ++i)
+            {
+                ReturnErrorOnFailure(encoder.Encode(mCluster->eventList[i]));
```

Comment:
It just occurred to me: for bridged nodes we have to have a way to do this.  Right now the macros bridges use to set up their dynamic endpoints don't allow passing in an event list.  As a followup (not part of this PR), we should add a way to do it... especially because for Bridged Device Basic Information this event is in fact required.

Reply:
#24907 to follow up